packages/bitcore-node/src/providers/chain-state/internal/internal.ts

```diff
@@ -338,9 +338,9 @@ export class InternalStateProvider implements IChainStateService {
     const allMissingAddresses = new Array<string>();
     let totalMissingValue = 0;
     const missingStream = cursor.pipe(
-      through2(
-        { objectMode: true },
-        async (spentCoin: MongoBound<ICoin>, _, done) => {
+      new Transform({
+        objectMode: true,
+        async transform(spentCoin: MongoBound<ICoin>, _, next) {
           if (!seen[spentCoin.spentTxid]) {
             seen[spentCoin.spentTxid] = true;
```

Comment:
For what it's worth - it seems strange to negative the conditional that got us to this block first thing. I think this belongs at the bottom of the conditional block, since presumably the work done is what marks 'seen' as true.